### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by DogeYeet**
Fixed style for main.py, main.py, 1_load_data.py, and 2_training_and_prediction.py based on pylint suggestions.

### CODE REVIEW

File: senbace/pages/1_load_data.py [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
-import streamlit as st
-import pandas as pd
-import numpy as np
 import os
 import re
+import streamlit as st
+import pandas as pd
 import matplotlib.pyplot as plt
 from prediction_and_training import make_prediction
+<<<<<<< HEAD
```
Comment (hanbincho) [inline senbace/pages/1_load_data.py:12]: Remove merge conflicts, specifically lines 12-15 and 19

File: senbace/pages/2_training_and_prediction.py [hunk 2]
```diff
@@ -62,20 +65,21 @@
...
 # When user presses button, create dataloader and start training
 if train_pressed:
     training_dataloader = make_prediction.load_data(download_image_path, int(user_batch_size), \
         download_score_path+score_file.name)
 
-
     st.write("Data loader was created!")
-
     # Facing some kind of import issue here...
+<<<<<<< HEAD
```
Comment (hanbincho) [inline senbace/pages/2_training_and_prediction.py:75]: Remove additional merge conflicts, lines 75-78 and 81

**Review by UzoUwaoma — CHANGES_REQUESTED** [review]
load_Data.py and training_and_prediction have some merge conflicts annotation in them. 
These are causing the tests to fail.

Can you remove those merge conflict annotations? e.g. <<<<<<< HEAD, =========

**Review by hanbincho — CHANGES_REQUESTED** [review]
Other than the remnants of the merge conflicts, it seemed like there was also a failed check in 2_training_and_prediction.py. Wasn't sure if that's because of the merge conflicts or a different issue.

**Pull request closed without merging**